uwmmss0348.xml

```diff
@@ -38,18 +37,19 @@
 	<archdesc level="collection" relatedencoding="MARC21">
 		<did>
 			<physloc type="locationcode" audience="internal">wimiugma</physloc>
-			<unittitle encodinganalog="245$a">
-				<!--Student Wisconsin Education Association Records -->
-				<unitdate type="inclusive" encodinganalog="245$f" normal="[ie. 1917/1940]"
-					><!--2000  to 2011--></unitdate>
+			<unittitle encodinganalog="245$a"> Student Wisconsin Education Association at
+				UW-Milwaukee Records, <unitdate type="inclusive" encodinganalog="245$f"
+					normal="2000/2011">2000-2011</unitdate>
 			</unittitle>
 			<origination label="Creator">
-				<corpname encodinganalog="110"><!--Student Wisconsin Education Association--></corpname>
-			</origination>		
-			<unitid label="Call Number" encodinganalog="099" repositorycode="us-wimiugma" countrycode="us"
-				>UWM Manuscript Collection 348</unitid>
+				<corpname encodinganalog="110">Student Wisconsin Education Association at
+					UW-Milwaukee</corpname>
+			</origination>
+			<unitid label="Call Number" encodinganalog="099" repositorycode="us-wimiugma"
+				countrycode="us">UWM Manuscript Collection 348</unitid>
 			<physdesc label="Quantity">
-				<extent encodinganalog="300"><!--1.6 cubic feet--> cubic ft. (3 boxes)</extent>
+				<extent encodinganalog="300">1.6 cubic feet (3 boxes)</extent>
+				<extent encodinganalog="300">147 digital files (9 GB)</extent>
```

Comment:
Added digital extent. There are a few ways to determine this which we can go over on Friday.